ServerCore/Pages/Puzzles/Create.cshtml

```diff
@@ -5,7 +5,7 @@
     ViewData["Title"] = "Create puzzle";
     ViewData["AdminRoute"] = "/Puzzles/Create";
     ViewData["AuthorRoute"] = "/Puzzles/Create";
-    ViewData["PlayRoute"] = "/Teams/Play";
+    ViewData["PlayRoute"] = "/Puzzles/PlayPuzzles";
```

Comment:
Any reason this can't be renamed to "/Puzzles/Play"?

Reply:
Fixed in newest iteration